.github/workflows/changelog.yml

```diff
@@ -0,0 +1,94 @@
+name: changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
```

Comment:
Which types are we excluding here? Can't we just use `on: pull_request`?

Reply:
I took this over from Pierre's initial PR for checking the existence of a changelog entry. The reasoning was that with the default configuration the pipeline is not re-triggered when a label is applied. But we need that, because adding/removing the "Skip news" label must trigger a new run of this check.